frontend/src/app/commonComponents/Select.tsx

```diff
@@ -23,6 +24,7 @@ interface Props<T> {
   selectClassName?: string;
   disabled?: boolean;
   className?: string;
+  registrationProps?: UseFormRegisterReturn<any>;
```

Comment:
for my understanding: what would go into typing the generic passed into `UseFormRegisterReturn` more explicitly (ie not using `any` here)? Vaguely understand that the set of input rules for the relevant components would be different and therefore difficult to type more explicitly, but trying to grok how the type system is interacting with the props here.